File: src/api/MaterialAdvisor/MaterialAdvisor.Application/Models/Shared/LanguageText.cs
using MaterialAdvisor.Data.Enums;

namespace MaterialAdvisor.Application.Models.Shared;

public class LanguageText
{
    public Guid Id { get; set; }

    public Language LanguageId { get; set; }

    public string Text { get; set; } = null!;
}

File: src/app/material-advisor/src/app/shared/models/LanguageText.ts
import { LanguageEnum } from "@shared/types/LanguageEnum";

export class LanguageText {
    text: string;
    languageId: LanguageEnum;

    constructor(text: string, languageId: LanguageEnum) {
        this.text = text;
        this.languageId = languageId;
    }
}

